src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -23,6 +22,7 @@
     <ILCompilerRuntimeIdentifier Condition="'$(TargetsWindows)' == 'true'">win-$(TargetArchitecture)</ILCompilerRuntimeIdentifier>
     <ILCompilerRuntimeIdentifier Condition="'$(TargetsLinux)' == 'true'">linux-$(TargetArchitecture)</ILCompilerRuntimeIdentifier>
     <ILCompilerRuntimeIdentifier Condition="'$(TargetsOSX)' == 'true'">osx-$(TargetArchitecture)</ILCompilerRuntimeIdentifier>
+    <ILCompilerRuntimeIdentifier Condition="'$(__DistroRid)' == 'linux-musl-x64'">linux-musl-x64</ILCompilerRuntimeIdentifier>
```

Comment:
It seems `ILCompilerRuntimeIdentifier` always equals to `__DistroRid`.  We can remove the duplicate variable then.

Reply:
Good idea. #777